sdk/iot_hub/src/service/mod.rs

```diff
@@ -816,6 +816,25 @@ impl ServiceClient {
         DeleteConfigurationBuilder::new(self.clone(), if_match.into(), configuration_id.into())
     }
 
+    /// Send a cloud-to-device message.
+    ///
+    /// ```
+    /// use azure_iot_hub::service::ServiceClient;
+    /// use std::collections::HashMap;
+    ///
+    /// # let connection_string = "HostName=cool-iot-hub.azure-devices.net;SharedAccessKeyName=iot_hubowner;SharedAccessKey=YSB2ZXJ5IHNlY3VyZSBrZXkgaXMgaW1wb3J0YW50Cg==";
```

Comment:
This secret must be removed and you should rotate it immediately. Do not put secrets in the repo. Mark this code fence `no_run` and use environment variables in your example.

Reply:
Use the same ones as your other example.

Reply:
It's not a real secret, if you base64 decode it you'll realize.

Reply:
Our secret scanner may still flag it. It doesn't decode the values (most won't be ASCII anyway). Since the value won't work anyway, you could just leave it blank.